undertow/src/main/java/org/wildfly/extension/undertow/deployment/UndertowDeploymentService.java

```diff
@@ -106,6 +106,9 @@ public void startContext() throws ServletException {
                 HttpHandler handler = deploymentManager.start();
                 Deployment deployment = deploymentManager.getDeployment();
                 host.get().registerDeployment(deployment, handler);
+            } catch (Throwable e ) {
```

Comment:
@aogburn I don't understand how this compiles in CI but this method can't throw Throwable. This would need to be catch (RuntimeException | ServletException e).

Reply:
Never mind; my IDE was complaining about this yesterday but seems to have gotten over it. The try block can only throw RuntimeException, ServletException or Error so the compiler allows it.